Formula/px4-sim-gazebo.rb

```diff
@@ -27,13 +14,13 @@ class Px4SimGazebo < Formula
   depends_on "gst-plugins-ugly"
   depends_on "gstreamer"
   depends_on "opencv"
-  depends_on "osrf/simulation/gazebo11"
+  depends_on "osrf/simulation/gz-harmonic"
   depends_on "protobuf"
   depends_on "px4-dev"
```

Comment:
is this not a circular dependency?

Reply:
we already have it for gazebo classic. I do not think if it is circulatory!